test/e2e/specs/scaleupdown.go

```diff
@@ -0,0 +1,177 @@
+package specs
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/sirupsen/logrus"
+	apiappsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	internalapi "github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/fakerp"
+	"github.com/openshift/openshift-azure/pkg/util/randomstring"
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
+)
+
+var _ = Describe("Scale Up/Down E2E tests [ScaleUpDown][Real][LongRunning][EveryPR]", func() {
```

Comment:
This test should work both for real and fake, right? Also, I am not sure we should be running it on every PR for now.

Reply:
20m on top of the current test runtime is a lot without running ginkgo tests in parallel. Also, it makes no sense to test every PR when the real RP is targetted.